src/modules/landing_target_estimator/KFxyzCoupledMoving.cpp

```diff
@@ -0,0 +1,483 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2013-2018 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/*
+ * @file KFxyzCoupledMoving.h
+ * Simple Kalman Filter for static target
+ *
+ * @author Jonas Perolini <jonas.perolini@epfl.ch>
+ *
+ */
+
+#include "KFxyzCoupledMoving.h"
+
+namespace landing_target_estimator
+{
+
+void KFxyzCoupledMoving::predictState(float dt, matrix::Vector<float, 3> acc)
+{
+	// Total ops: 32
+
+	const float tmp0 = 0.5f * dt * dt;
+
+	_state(0, 0) = _state(0, 0) + _state(3, 0) * dt + _state(9, 0) * tmp0 - tmp0 * acc(0);
+	_state(1, 0) = _state(1, 0) + _state(10, 0) * tmp0 + _state(4, 0) * dt - tmp0 * acc(1);
+	_state(2, 0) = _state(11, 0) * tmp0 + _state(2, 0) + _state(5, 0) * dt - tmp0 * acc(2);
+	_state(3, 0) = _state(3, 0) + _state(9, 0) * dt - acc(0) * dt;
+	_state(4, 0) = _state(10, 0) * dt + _state(4, 0) - acc(1) * dt;
+	_state(5, 0) = _state(11, 0) * dt + _state(5, 0) - acc(2) * dt;
+	// _state(6, 0) = _state(6, 0);
+	// _state(7, 0) = _state(7, 0);
+	// _state(8, 0) = _state(8, 0);
+	// _state(9, 0) = _state(9, 0);
+	// _state(10, 0) = _state(10, 0);
+	// _state(11, 0) = _state(11, 0);
```

Comment:
it's great to use SymForce, but you need to leave the generated code in the generated .h file and not manually modified

Reply:
Done in https://github.com/PX4/PX4-Autopilot/pull/20653/commits/dbe925698099776817ebab00d4d237da2ff21384